src/main/resources/com/microsoft/azure/util/AzureCredentials/credentials.jelly

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:st="jelly:stapler" xmlns:d="jelly:define">
+    <j:set var="uniqueId" value="${h.generateId()}" />
+    <f:entry title="${%Subscription ID}" field="subscriptionId" help="/plugin/azure-vm-agents-plugin/help-subscriptionId.html">
+      <f:password />
+    </f:entry>
+    <f:entry title="${%Client ID}" field="clientId" help="/plugin/azure-vm-agents-plugin/help-clientId.html">
+      <f:password />
+    </f:entry>
+    <f:entry title="${%Client Secret}" field="clientSecret" help="/plugin/azure-vm-agents-plugin/help-clientSecret.html">
+      <f:password />
+    </f:entry>
+    <f:entry title="${%OAuth 2.0 Token Endpoint}" field="oauth2TokenEndpoint" help="/plugin/azure-vm-agents-plugin/help-oauth2TokenEndpoint.html">
+      <f:password />
+    </f:entry>
+    <f:entry title="${%Management Service URL}" field="serviceManagementURL" help="/plugin/azure-vm-agents-plugin/help-serviceManagementURL.html">
+      <f:textbox default="${descriptor.getDefaultserviceManagementURL()}"/>
+    </f:entry>
+    <st:include page="id-and-description" class="${descriptor.clazz}"/>
+    <f:validateButton title="${%Verify Configuration}" progress="${%Verifying}" method="verifyConfiguration" 
+          with="subscriptionId,clientId,clientSecret,oauth2TokenEndpoint,serviceManagementURL" />
+</j:jelly>
```

Comment:
What's the purpose of this file?  I thought we're now loading all the credentials from the credential store?